renderers/rendererfactory.go

```diff
@@ -0,0 +1,22 @@
+package renderers
```

Comment:
Should this line say "package render" instead of the plural? 

https://pkg.go.dev/github.com/unrolled/render